emmet-game/src/css/Style.css

```diff
@@ -0,0 +1,106 @@
+/* common */
+
+body {
+    display: grid;
+    grid-template-rows: 1fr 1fr 1fr;
+    grid-template-columns: 1fr;
+    height: 100vh;
+}
+h1 {
+    font-size: 1em;
+    color: white;
+}
+nav {
+    display: grid;
+    grid-template-areas: "home title play";
+    grid-template-rows: 40px;
+    grid-template-columns: 1fr 4fr 1fr;
+    text-align: center;
+}
+
+.nav__home {
+    grid-area: home;
+}
+.nav__title {
+    grid-area: title;
+}
+.nav__start {
+    grid-area: play;
+}
+
+main {
+    background: linear-gradient(
+        0deg,
+        rgba(75, 192, 215, 1) 9%,
+        rgba(92, 45, 253, 1) 100%
+    );
+    height: 100vh;
+    width: 100vw;
+    font-size: calc(10px + 2vmin);
+    color: #555;
+}
+
+/* StartArea.js*/
+
+.startarea__main {
+    text-align: center;
+    width: 80%;
+    margin: 30% auto;
+}
+
+/* GameArea.js*/
```

Comment:
Hat es einen Grund, warum du keine styled components verwendest? Wäre es nicht einfacher, wenn dein css direkt in der zugehörigen Komponente wäre?